conf/messages

```diff
@@ -583,11 +584,13 @@ pages.officerContact.emailAddress.label                  = Email address
 pages.officerContact.mobile.label                        = Mobile number
 pages.officerContact.daytimePhone.label                  = Phone number
 validation.officerContact.missing                        = Enter your contact details
-validation.officerContact.atLeastOneContact.missing                        = Enter your contact details
-validation.officerContactDetails.mobile.invalid          = Enter a valid contact number
-validation.officerContactDetails.daytimePhone.invalid    = Enter a valid contact number
+validation.officerContact.atLeastOneContact.missing      = Enter your contact details
 validation.officerContactDetails.email.invalid           = Enter a valid email address
 validation.officerContactDetails.email.maxlen            = Email address must not exceed 70 characters
+errors.invalid.officeContactNum.tooLong                  = Enter a contact number using 20 digits or less
+errors.invalid.officeContactNum.tooShort                 = Enter a contact number using 10 digits or more
+errors.invalid.officeContactNum                          = Enter a valid contact number
```

Comment:
again why have these changed format from validation to errors

Reply:
We could make these generic so you can make your isvalid phonenumber generic and not have to pass any strings in containing the form name. and it would only be in the messages file once